fixfrontend-reference-code.js

```diff
@@ -0,0 +1,997 @@
+ import { flushSync } from 'react-dom';
+import logo from './logo.svg';
+import './App.css';
+import Layout from "./pages/layout";
+import Home from "./pages/index";
+import Blogs from "./pages/blogs";
+import Contact from "./pages/contact";
+import SingIn from "./pages/signin";
+import SignUp from "./pages/signup";
+import Error404  from "./pages/error404";
+
+function App() {
+
+
+
+
+//View Meeting
+const viewMyMeeting = async(e)=>{
+//e.preventDefault()	
+
+fetch("/w/viewevent/9hs1c70CG",{
+	
+method: 'GET',
+    headers: {
+        'Content-Type': 'application/json; charset=UTF-8'
+    },
+
+
+}) .then((response) => response.json())
+   .then((data) => {
+     // console.log(data);
+      setMeeting_shedule_data([data]);
+      /*
+	  setSignUpInfo([data].map((info)=>{ return [info._id,info.meeting_title,info.usser_meeting_data,
+	  info.meeting_url,info.created_on,info.meeting_event_creator,info.date_duration,info.time,info.time_gap]}));
+	  */
+	  setSignInInfo([data].map((info)=>{ return [info._id,info.meeting_title,info.usser_meeting_data,
+	  info.meeting_url,info.created_on,info.meeting_event_creator,info.date_duration,info.time,info.time_gap]}))
+   })
+   .catch((err) => {
+      console.log(err.message);
+   });
+
+}
+
+
+
+useEffect(()=>{
+	
+viewMyMeeting()
+	
+},[])
+
+//MEETING STATE	
+const [meeting_shedule_data, setMeeting_shedule_data] = useState([])
+
+//USER TEMPLATE STATE
+const [user_meeting_shedule_data_template, setUser_meeting_shedule_data_template] = useState([])
+
+
+//console.log(user_meeting_shedule_data_template)	
+
+	
+	/////////////// SIGN UP ////////////// DONT USE!!!!!
+	const [data, setData] = useState([]);
+	 const [data1, setData1] = useState();
+	 const [signUpMessage, setSignUpMessage] = useState('');
+	 
+	 const [signUpInfo, setSignUpInfo] = useState({});
+
+	 const signupButton = useRef(null)
+	 
+	
+const handleSubmit = async(e)=>{
+e.preventDefault()	
+
+fetch("/w/signup",{
+	
+method: 'POST',
+    headers: {
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+        user: signUpInfo
+    })
+
+
+}).then((signUpStatus)=>{console.log(signUpStatus);signUpStatus.status === 200 ? setSignUpMessage('Your Account has been sucessfully created') : setSignUpMessage('Could not complete user signup') })
+
+
+}
+
+const buttonAnimationSignUp = ()=>{
+	
+signupButton.current.style.marginRight = '5%';
+signupButton.current.style.transition = '00.2s'
+
+const endAnim = setTimeout(()=>	{
+signupButton.current.style.marginRight = '0%';
+signupButton.current.style.transition = '00.2s'
+},300)
+
+return ()=> clearTimeout(endAnim)
+	
+}
+
+const signUpEmail = (e)=>{	
+setSignUpInfo((info)=>({
+	
+...info,
+username: 	e.target.value
+	
+}))		
+}
+
+const signUpPass = (e)=>{	
+setSignUpInfo((info)=>({
+...info,
+pass: 	e.target.value
+	
+}))	
+ 	
+	
+}
+
+
+/////////////// SIGN IN
+
+	/// const [dataSignIn, setDataSignIn] = useState([]);
+	// const [data1SignIn, setData1SignIn] = useState();
```

Comment:
same here; does this need to be deleted?

Reply:
Don't use it cos, the reference site does not have a place for sign up.... Both the sign in and sign up are in the place..

Don't delete it either... Just ignore it..

Reply:
Don't delete... Thats for signing in...

Reply:
YES!